servicemigration/version.py

```diff
@@ -1,5 +1,6 @@
+import os
 
-API_VERSION = "1.0.0"
+API_VERSION = open(os.path.join(os.path.dirname(__file__), "VERSION"), 'r').readlines()[0]
```

Comment:
This won't work unless we ship the VERSION file in the docker image, which requires minor changes to makefile and Dockerfile.